apps/paymaster-proxy/docker-compose.yml

```diff
@@ -1,7 +1,12 @@
 version: '3.8'
 
+include:
+  - path: ../api-key-service/docker-compose.yml
```

Comment:
this will bring up all services with specified profile in the api-key-service docker compose